docs/documentation/cert-manager.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Cert-Manager
+description: Cert-Manager
+layout: docs
+permalink: /docs/cert-manager
+---
+### Cert-Manager
+
+Dekorate supports generating `Certificate` and `Issuer` resources using [Cert-Manager](https://cert-manager.io/). When these resources are loaded into the cluster, Cert-Manager will issue the certificates, issuers, and trust-store files that then can be used by final applications.
```

Comment:
"Dekorate supports generating `Certificate` and `Issuer` resources ..." ---> "Dekorate supports to generate a X.509 certificate with the help of the Certificate and Issuer CRD resources handled by the [Cert-Manager](https://cert-manager.io/)."

AND

"When these resources are loaded into the cluster, Cert-Manager" --> "When these CRD resources are deployed on the cluster, the Cert-Manager will process them in order to populate a Secret containing by example a: CA certificate, private key, server certificate or java keystores, etc"